org.eclipse.xtext/src/org/eclipse/xtext/formatting/impl/FormattingConfigBasedStream.java

```diff
@@ -269,7 +269,9 @@ public LineEntry(EObject grammarElement, String value, boolean isHidden, Set<Ele
 			this.isHidden = isHidden;
 			this.leadingLocators = beforeLocators;
 			this.indent = indent;
-			this.leadingWS = leadingWS;
+			if (leadingWS != null) {
```

Comment:
I'd rather like to fix the caller of the constructor.

Reply:
Done